### PR TITLE
- Some of the new structs missing in 2.24

### PR DESCRIPTION
**Pull request by HolterPhylo**

### CODE REVIEW

File: libMBIN/Source/NMS/GameComponents/GcDeathQuote.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class GcDeathQuote : NMSTemplate
         public string QuoteLine1;
         [NMS(Size = 0x80)]
         public string QuoteLine2;
-        [NMS(Size = 0x80)]
+        [NMS(Size = 0x20)]
```
Comment (monkeyman192) [inline libMBIN/Source/NMS/GameComponents/GcDeathQuote.cs:15]: Whoops! Thanks for picking this up!

File: libMBIN/Source/NMS/GameComponents/GcSolarGenerationData.cs [hunk 2]
```diff
@@ -5,8 +5,9 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-    [NMS(Size = 0x8, GUID = 0xEBECDBDBC0F9EFB9, NameHash = 0x62C0729D49C7216D, Broken = true)]
+    [NMS(Size = 0x8, GUID = 0xEBECDBDBC0F9EFB9, NameHash = 0x62C0729D49C7216D)]
     public class GcSolarGenerationData : NMSTemplate
     {
+        /* 0x00 */ public float SolarSeed;
```
Comment (monkeyman192) [inline libMBIN/Source/NMS/GameComponents/GcSolarGenerationData.cs]: should this not be a `ulong` type?

Comment (HolterPhylo) [reply libMBIN/Source/NMS/GameComponents/GcSolarGenerationData.cs]: Whoops! myself, you are right

File: libMBIN/Source/NMS/Toolkit/TkHeavyAirSystem.cs [hunk 3]
```diff
@@ -5,8 +5,25 @@
...
     public class TkHeavyAirSystem : NMSTemplate
     {
+        [NMS(Size = 0x80)]
+        /* 0x000 */ public string Material;
+        /* 0x080 */ public Vector3f MajorDirection;
+        /* 0x090 */ public Vector3f ScaleRange;
+        /* 0x0A0 */ public Vector3f RotationSpeedRange;
+        /* 0x0B0 */ public Vector3f FadeSpeedRange;
+        /* 0x0C0 */ public Vector3f TwinkleRange;
+        /* 0x0D0 */ public Vector3f AmplitudMin;
```
Comment (monkeyman192) [inline libMBIN/Source/NMS/Toolkit/TkHeavyAirSystem.cs]: Typo? (`Amplitude`)? (and next line...

Comment (HolterPhylo) [reply libMBIN/Source/NMS/Toolkit/TkHeavyAirSystem.cs]: Yes, missing 'e' both

Comment (HolterPhylo) [reply libMBIN/Source/NMS/Toolkit/TkHeavyAirSystem.cs]: corrected

**Pull request merged**